lib/smf/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+config SMF
+	bool "Enable Hierarchical State Machine"
+	help
+	  This option enables the Hierarchical State Machine library
+
+if SMF
+
+config SMF_FLAT
+	bool "Restrict to Flat State Machine"
+	help
+	  Enabling this option will not allow any parent states, resulting
+	  in a Flat State Machine.
+
+config SMF_PRINT_MSG
```

Comment:
isn't logging sufficient? Should that option depend on logging being enabled?

Reply:
Yes I agree that the message printing is unnecessary because developers can use logging.

Reply:
@sambhurst I see that this option is not checked anywhere, is giving output about states to-be-implemented?
I smf.c the only LOG usage is 
`LOG_WRN("Calling %s from exit action", __func__);`